opentdf-dev.yaml

```diff
@@ -17,6 +17,16 @@ services:
       - kid: e1
         alg: ec:secp256r1
         legacy: true
+      - kid: e2
+        alg: ec:secp384r1
+      - kid: e2
+        alg: ec:secp384r1
+        legacy: true
+      - kid: e3
+        alg: ec:secp521r1
```

Comment:
Can you add a variable to the `encrypt` that is like `--algorithm` or `--curve` so we can use these in the bats test?